test/e2e/framework/jobs_util.go

```diff
@@ -269,10 +269,8 @@ func newBool(val bool) *bool {
 	return p
 }
 
-type updateJobFunc func(*batch.Job)
```

Comment:
Seems like we can do more than just this for jobs_utils :). What are your thoughts to moving the code here to a dedicated package `test/e2e/framework/jobs`?

Reply:
That sounds like a good idea. We could split the original large file into smaller one. I'll work on to it to see how it goes. :)